org-roam.el

```diff
@@ -586,12 +586,12 @@ it as FILE-PATH."
           (let* ((type (org-roam--collate-types (org-element-property :type link)))
                  (path (org-element-property :path link))
                  (element (org-element-at-point))
```

Comment:
as I explained in comment, it seems that `element` might be `nil`, though I am not sure if it's valid case or not. So in my local copy I monkey-patched this function by changing `let*` to `when-let*`. WDYT, should I include that change here as well?